toolkits/global/packages/global-corporate-footer/README.md

```diff
@@ -71,21 +71,21 @@ If you're still not sure, email the Data Protection Manager, Quinton Creighton,
 
 The extended footer contains the same links as the default footer, with the option to include:
 
-- California privacy statement
+- Your US state privacy rights
 - Terms and conditions
 - Help and support
 
-#### California privacy statement
+#### Your US state privacy rights
 
-Your product or service needs to include a California privacy statement if it meets one of the following criteria.
+Your product or service needs to include a Your US state privacy rights if it meets one of the following criteria.
```

Comment:
Replacing `California privacy statement` with `Your US state privacy rights` might be making this part unclear (this also exists elsewhere in the PR when the link name is mentioned within body of text). 

Im not sure what the style should be but maybe some formatting of the link name is needed.